.gitignore

```
@@ -30,19 +128,58 @@ ENV/
 env.bak/
 venv.bak/
 
+# Spyder project settings
+.spyderproject
+.spyproject
+
+# Rope project settings
+.ropeproject
+
+# mkdocs documentation
+/site
+
 # mypy
 .mypy_cache/
 .dmypy.json
 dmypy.json
 
-token.txt
-src/pyosmeta/_version.py
+# Pyre type checker
+.pyre/
+
+# pytype static type analyzer
+.pytype/
+
+# Cython debug symbols
+cython_debug/
 
-# local files
+# PyCharm
+#  JetBrains specific template is maintained in a separate JetBrains.gitignore that can
+#  be found at https://github.com/github/gitignore/blob/main/Global/JetBrains.gitignore
+#  and can be added to the global gitignore or merged into this file.  For a more nuclear
+#  option (not recommended) you can uncomment the following to ignore the entire idea folder.
+.idea/
+
+# macOS
+.DS_Store
+
+# vscode editor
+.vscode/
+
+# pyosMeta specific
+all_contribs.pickle
+all_contribs_dict.pickle
 all-submissions.csv
 issue-stats.ipynb
 plot.py
 presubmissions.csv
 reviews_all.csv
 reviews_presubmissions.csv
+src/pyosmeta/_version.py
 src/pyosmeta/cli/test.ipynb
+
+contrib*.yml
+pack*.yml
+*.pickle
```

```suggestion
src/pyosmeta/_version.py
*.pickle
```

This is my mess 😄 i have learned to use a global .gitignore for dummy test files :)  so we can clean this up too as i have a better workflow now!